docs/minutes/20201010.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: minutes
+date: 2020-10-10
+---
+
+_\*Foreword: In lieu of CS2100 midterm assessment, the meeting was postponed to 10 October 2020._
+
+### Updated Architecture
+
+Issue: Need to support UI-dependent behaviour for commands.
+
+Solution: Pass the UI state to the command parser.
+
+### Actionables
+
+Wei Liang:
+- Create {Add, Delete} × {Expense, Income} Command
+  - Generic AddCommand that works based on the current UiState
```

Comment:
Possible to add backticks to parts which reference code? i.e. `AddCommand`

Reply:
Fixed